test/request-signal.js

```diff
@@ -36,9 +36,11 @@ test('post abort signal', async (t) => {
 
   server.listen(0, async () => {
     const ac = new AbortController()
-    const ures = await request(`http://0.0.0.0:${server.address().port}`, { signal: ac.signal })
+    const uresPromise = request(`http://0.0.0.0:${server.address().port}`, { signal: ac.signal })
```

Comment:
ℹ️ this test was wrong because `request` api terminates when the stream has been completely consumed, so the connection has been closed. If we postpone the `await` we can throw the abort before dispatch starts consuming the stream, so the request is actually aborted